### DISCUSSION TITLE
Gordon

### DISCUSSION THREAD

**Discussion by amytimed** [qhdwight/bevy_fps_controller#26] [General]
Gordon! She's gone Gordon. She's gone. SON OF A GIT AND HIS "UNFORESEEN CONSEQUENCES"! I knew it! When I get my hands on him, I'm gonna... I'm gonna kill him.

Come on Gordon. We've got work to do.

**Comment by amytimed** [comment]
great source engine inspired fps controller, it makes me want to make Half-Life Bevy

**Reply by amytimed** [reply]
Half-Life 1: Source: Bevy

**Reply by qhdwight** [reply]
Excellent content thank you

**Reply by qhdwight** [reply]
Perhaps you should make it 👀

**Reply by amytimed** [reply]
yes

**Reply by qhdwight** [reply]
Also in case you are interested, there was recently a 25th anniversary documentary made about it: https://youtu.be/TbZ3HzvFEto?si=SOl8A9NfWd9rZGw1

**Reply by amytimed** [reply]
yes
